src/components/KonnectorUpdateInfo/styles.styl

```
@@ -0,0 +1,4 @@
+.KonnectorUpdateInfo__message a
+    font-weight bold
+    text-decoration none
+    color dodgerBlue
```

can you use `var(--primary)`
```suggestion
     color var(--primary)
```

---

Oh yes. And I have other hard coded colors that I forgot to manage. Thanks for the headsup